### DISCUSSION TITLE
Windows 7/8x support

### DISCUSSION THREAD

**Discussion by ericoepico69** [zapSNH/zapsCoolPhotonTheme#10] [Ideas]
![2017-11-10-18-34-51-4d3149](https://github.com/user-attachments/assets/d40b3151-974b-4435-9b8a-69c246baaebc)

i would love to see this

**Comment by zapSNH** [comment]
IIRC the theme worked pretty well with 7, not sure about 8 though...
You can download the last version for Firefox 115 here if you want: https://github.com/zapSNH/zapsCoolPhotonTheme/releases/tag/v128
I'm not really planning on re-supporting 115esr since it would be a hassle to port and fix up the latest version of the theme for 115esr.

If you're talking about support win 7/8.x with FF versions past 115 (like [r3dfox](https://github.com/Eclipse-Community/r3dfox)), I _might_ add official support for it. (although even w/o official support it should be fine with a couple fixes in the customChrome files)
